pczt/src/lib.rs

```diff
@@ -40,6 +48,10 @@ const MAGIC_BYTES: &[u8] = b"PCZT";
 const PCZT_VERSION_1: u32 = 1;
 
 /// A partially-created Zcash transaction.
+///
+/// The PCZT format extends the PSBT format ([BIP 174], [BIP 370]) to support Zcash's
```

Comment:
This is incorrect; while it is analogous to PSBT, it is explicitly *not* an extension.